pkgs/development/python-modules/openai-whisper/default.nix

```diff
@@ -39,13 +45,30 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [
     numpy
-    torch
     tqdm
     more-itertools
     transformers
     ffmpeg-python
+    numba
+    scipy
+    tiktoken
+  ] ++ lib.optionals (!cudaSupport) [
+    torch
+  ] ++ lib.optionals (cudaSupport) [
+    openai-triton
+    torchWithCuda
   ];
 
+  postPatch = ''
+    sed -i 's/tiktoken==0.3.1/tiktoken>=0.3.1/' requirements.txt
```

Comment:
Ideally using `substituteInPlace`, so we get a log entry when it stops applying.

Reply:
Done!👍

> Ideally using `substituteInPlace`, so we get a log entry when it stops applying.